api/v1alpha1/etcdcluster_types.go

```diff
@@ -16,13 +16,20 @@ type EtcdClusterSpec struct {
 	Storage *EtcdPeerStorage `json:"storage,omitempty"`
 }
 
+type EtcdMember struct {
+	Name string `json:"name"`
+	ID   string `json:"ID"`
+}
+
 // EtcdClusterStatus defines the observed state of EtcdCluster
 type EtcdClusterStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	// Members contains information about each member from the etcd cluster.
+	// +optional
```

Comment:
nit I was about to say that it should be `Etcd`, but I see that the project readme talks about `etcd`, so I'll have to update it in all the docs that I've written.